src/node_crypto.cc

```diff
@@ -3065,12 +3092,15 @@ void CipherBase::Init(const char* cipher_type,
 
 void CipherBase::Init(const FunctionCallbackInfo<Value>& args) {
   CipherBase* cipher = Unwrap<CipherBase>(args.Holder());
+  Environment* env = cipher->env();
 
-  if (args.Length() < 2 ||
-      !(args[0]->IsString() && Buffer::HasInstance(args[1]))) {
-    return cipher->env()->ThrowError("Must give cipher-type, key");
+  if (args.Length() < 2) {
+    return env->ThrowError("Cipher type and Key arguments are mandatory");
```

Comment:
@bnoordhuis `cipher-type`. Should that be `cipher-type` itself or `Cipher type`?